packages/components/src/__actions__/Menu/v3/Menu.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import {
+  Menu as RACMenu,
+  MenuProps as RACMenuProps,
+} from "react-aria-components"
+import styles from "./Menu.module.scss"
+
+export type MenuProps = Omit<
+  // @todo: what does it want here instead of any?
+  RACMenuProps<any>,
```

Comment:
if it's a generic you can make this type generic as well and pass it through.

something like this, but replace the any with the type that RACMenuProps expects
```ts
export type MenuProps<T extends any> = Omit<
  RACMenuProps<T>,
```

Reply:
I've ended up baking `HTMLDivElement` into this, I'm not sure why it would be anything else